practice.js

```diff
@@ -21,7 +24,12 @@ var moreFruits = function(fruits) {
 
 // use _.each to traverse the number array and determine
 // which are multiples of five.
-var multiplesOfFive = function(numbers) {
+var multiplesOfFive = function (numbers) {
+
+};
+
+// use _.each to traverse the tweets array to build a user specific array.
+var getUserTweets = function(tweets, user) {
 
 };
```

Comment:
Can we word this more like "build an array containing only tweets belonging to a specified user"?